relay-general/src/protocol/event.rs

```diff
@@ -468,6 +468,11 @@ pub struct Event {
     #[metastructure(omit_from_schema)] // we only document error events for now
     pub spans: Annotated<Array<Span>>,
 
+    /// Measurements
+    #[metastructure(skip_serialization = "empty_deep")]
```

Comment:
`empty_deep` is a rather expensive check, would `empty` be enough here?

Reply:
done in https://github.com/getsentry/relay/pull/724/commits/9c58a34104caec3c704a7e01b6198bd020289b90